service/mantle/training/TrainingServices.xml

```diff
@@ -0,0 +1,136 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<services xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+          xsi:noNamespaceSchemaLocation="http://moqui.org/xsd/service-definition-2.1.xsd">
+
+
+    <service verb="create" noun="Product" displayName="Create a product" allow-remote="true">
+        <description>To add details of new product in respective table</description>
+        <in-parameters>
+            <parameter name="productName" required="true"/>
+            <parameter name="price" required="true"/>
+            <parameter name="description"/>
+            <parameter name="priceUomId" default-value="USD"/>
+        </in-parameters>
+        <out-parameters>
+            <parameter name="productId"/>
+
+        </out-parameters>
+        <actions>
+            <service-call name="create#mantle.product.Product" in-map="context" out-map="context"/>
+
+            <service-call name="create#mantle.product.ProductPrice" in-map="context + [productId:productId]"
+                          out-map="context"/>
+        </actions>
+    </service>
+
+
+     <service verb="get" noun="ProductOne">
+        <description>To get the details of a specific</description>
+        <in-parameters>
+            <parameter name="productId" required="true"/>
+        </in-parameters>
+        <out-parameters>
+            <parameter name="productId"/>
+            <parameter name="description"/>
+            <parameter name="productName"/>
+            <parameter name="price"/>
+        </out-parameters>
+        <actions>
+            <entity-find-one entity-name="mantle.product.Product" value-field="product"/>
+            <set field="productId" from="product.productId"/>
+            <set field="description" from="product.description"/>
+            <set field="productName" from="product.productName"/>
+            <entity-find-one entity-name="mantle.product.ProductPrice" value-field="productPrice"/>
+            <set field="price" from="productPrice.price"/>
+            <set field="priceUomId" from="productPrice.priceUomId"/>
+        </actions>
+    </service>
+
+
+    <service verb="get" noun="Product">
+        <description>To get details of all the products</description>
+
+        <out-parameters>
+            <parameter name="products"/>
+        </out-parameters>
+        <actions>
+
+            <entity-find entity-name="Product" list="productDetailList">
+                <select-field field-name="productId"/>
+                <select-field field-name="productName"/>
+                <select-field field-name="description"/>
+            </entity-find>
+            <entity-find entity-name="ProductPrice" list="productPriceList">
```

Comment:
Can you please update code as per our discussion? We should not fetch the product and price information separately. We need to find the related price of the product.

Reply:
I think we don't need productPriceList here. Please update.